internal/repository/get_stats.go

```diff
@@ -0,0 +1,30 @@
+package repository
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+
+	"github.com/dangerousmonk/short-url/internal/models"
+)
+
+// GetStats returns total number of stored URLs and users
+func (r *PostgresRepo) GetStats(ctx context.Context) (models.InternalStats, error) {
+	var stats models.InternalStats
+
+	query := `
+	SELECT 
+    	COUNT(*) AS urls,
+    	COUNT(DISTINCT user_id) AS users
+	FROM urls;`
```

Comment:
Здесь стоит строку запроса сделать именованной константой. 
```suggestion

const getStatsQuery = `
	SELECT 
		COUNT(*) AS urls,
		COUNT(DISTINCT user_id) AS users
	FROM urls;`

// GetStats returns total number of stored URLs and users
func (r *PostgresRepo) GetStats(ctx context.Context) (models.InternalStats, error) {
	var stats models.InternalStats
```